docs/src/main/docs/guides/12_tracing_mp_guide.adoc

```diff
@@ -0,0 +1,759 @@
+
+=== Tracing across services
+
+Helidon automatically traces across services, providing that the services use the same tracer, for example, the same instance of Zipkin.
+This means a single trace can include spans from multiple services and hosts.  OpenTracing uses a `SpanContext` to
+propagate tracing information across process boundaries.  When you make client API calls, Helidon will
+internally call OpenTracing APIs to propagate the `SpanContext`. There is nothing you need to do in your application to make this work.
+
+To demonstrate distributed tracing, you will need to create a second project, where the server listens on port 8081.
+Create a new root directory to hold this new project, then do the following steps, similar to
+what you did at the start of this guide:
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-mp \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-mp \
+    -Dpackage=io.helidon.examples.quickstart.mp
+----
+
+[source,bash]
+.The project will be built and run from the `helidon-quickstart-mp` directory:
+----
+cd helidon-quickstart-mp
+----
+
+[source,xml]
+.Add the following dependency to `pom.xml`:
+----
+<dependency>
+  <groupId>io.helidon.tracing</groupId>
+  <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+----
+
+[source,bash]
+.Replace `META-INF/microprofile-config.properties` with the following:
+----
+app.greeting=Hello From MP-2
+tracing.service=helidon-mp-2
+
+# Microprofile server properties
+server.port=8081
+server.host=0.0.0.0
+----
+
+[source,bash]
+.Build the application, skipping unit tests, then run it:
+----
+mvn package -DskipTests=true
+java -jar target/helidon-quickstart-mp.jar
+----
+
+[source,bash]
+.Run the curl command in a new terminal window and check the response (*notice the port is 8081*) :
+----
+curl http://localhost:8081/greet
+...
+{
+  "message": "Hello From MP-2 World!"
+}
+----
+
+
+Once you have validated that the second service is running correctly, you need to modify the original application to
+call it.
+
+[source,java]
+.Update the `GreetResource` class with the following code:
+----
+package io.helidon.examples.quickstart.mp;
+
+import io.helidon.security.integration.jersey.SecureClient;
+import java.util.Collections;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import org.glassfish.jersey.server.Uri;
+
+@Path("/greet")
+@RequestScoped
+public class GreetResource {
+
+  @Uri("http://localhost:8081/greet")
+  @SecureClient
```

Comment:
SecureClient should not longer be used - it is deprecated in the current `master`, just remove it and your example will still work

Reply:
fixed

Reply:
I just tested without @SecureClient and it works fine.